docs/user/plagiarism-check.rst

```diff
@@ -172,6 +173,75 @@ The process is visualized in the following diagram.
 2. Reply to the message of the instructor.
 3. Wait for the final verdict.
 
+Continuous Plagiarism Control
+-----------------------------------------
+
+In this section, we explain the automated flow of the continuous plagiarism control (CPC) and present actions taken by different Artemis users.
+
+Automated Flow
+^^^^^^^^^^^^^^
+
+The CPC is an experimental feature designed to shorten the feedback loop for students and reduce the manual work required from tutors and instructors.
+Continuous feedback on potential plagiarism aims to encourage students to submit their original work and enhance their learning outcomes.
+The continuous plagiarism control **does not accuse students of plagiarism, nor does it affect their scores**.
+Instead, it **notifies students of significant similarities** detected between their submissions, allowing them to revise their solutions.
+
+The CPC initiates plagiarism checks every night for all onboarded exercises, processing all the latest submissions for each exercise.
+If a significant similarity is detected, the CPC creates or updates a significant similarity plagiarism case for that submission.
+It is a special form of plagiarism case, which is marked as a warning and presents a gentle message about detected similarity.
+If a submission does not include significant similarity, the CPC removes any associated existing significant similarity plagiarism case.
+Subsequently, the student analyzes the feedback received and, if the submission period is still open, can submit a new solution.
+If the submission period has ended, the student can either dispute the plagiarism case or choose to take no action.
+Eventually, the instructor makes a decision about accusing student of plagiarism and about any further steps.
+
+The following activity diagram depicts the CPC flow described above.
+
+|cpc-activity-diagram|
+
+Instructors
+^^^^^^^^^^^
+
+Configuration of the CPC
+""""""""""""""""""""""""
+
+Instructors can configure various aspects of the CPC for each exercise. The following list describes all configuration options:
+
+* **Instructors can enable the CPC.** - Instructor can turn the CPC on and off for a particular exercise. Default: `OFF`.
+* **Instructors can enable the post due date checks.** - Instructor can configure the CPC to test the exercise one more time the night after the exercise due date. This allows to test submission posted shortly before the due date. Default: `OFF`.
```

Comment:
The last sentence is very hard to understand. Does this refer to multiple submissions or is it passive tone for one submission (be posted)?

Reply:
It's describes a general case where the submissions are made close to the dead line. Changed to `This allows to test **submissions** posted shortly before the due date`.